core/shared/src/main/scala/zio/concurrent/ConcurrentMap.scala

```diff
@@ -27,6 +35,28 @@ final class ConcurrentMap[K, V] private (private val underlying: ConcurrentHashM
   def computeIfPresent(key: K, remap: (K, V) => V): UIO[Option[V]] =
     UIO(Option(underlying.computeIfPresent(key, remapWith(remap))))
 
+  /**
+   * Tests whether a given predicate holds true for at least one element in a map.
+   */
+  def exists(p: ((K, V)) => Boolean): UIO[Boolean] =
```

Comment:
I'd define predicate as `(K, V) => Boolean`, even though it's not aligned with Scala's specification.